korangar/src/input/mod.rs

```diff
@@ -45,6 +46,7 @@ pub struct InputSystem {
     mouse_input_mode: MouseInputMode,
     input_buffer: Vec<char>,
     picker_value: Arc<AtomicU64>,
+    previous_mouse_button: (MouseButton, ClientTick),
```

Comment:
Can you please include an intermediate struct that has descriptive names for the fields?